### PR TITLE
Extend base html

### PR DESCRIPTION
**Pull request by pengxuemoua**
Extend base html inheritance. 

Updated CSS

Center aligned the data on car_recalls.html.

Still a work in progress, but this is a step in the right direction. 

Sending for approval to merge into main. @Mr-3rd @amylynndee 

Pertaining to issue #45

### CODE REVIEW

**Comment by amylynndee** [comment]
@pengxuemoua the styling looks really good, I noticed the reset button is gone, are we going without it going forward?

File: static/cars_style.css [hunk 1]
```diff
@@ -39,3 +40,11 @@ h3 {
     margin-right: auto;
     display: block;
 }
+
+.recall_info {
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
```
Comment (Mr-3rd) [inline static/cars_style.css:66]: @pengxuemoua Can we change this to `display: table;`

This has all of the items connecting to their bullet points. My centered view was very distorted

**Review by Mr-3rd — CHANGES_REQUESTED** [review]
@pengxuemoua I have one small change so far, working on reviewing now

**Comment by amylynndee** [comment]
witnessed @Mr-3rd update code, everything looks good and works, we are good to merge!

**Pull request merged**